RadarSDK/RadarUser.m

```diff
@@ -95,6 +97,7 @@ - (instancetype _Nullable)initWithObject:(NSObject *)object {
     RadarLocationSource source = RadarLocationSourceUnknown;
     RadarTrip *trip;
     RadarFraud *fraud;
+    BOOL debug = YES;
```

Comment:
[Android seems to default to false here](https://github.com/radarlabs/radar-sdk-android/commit/7f7ffbe800ab08662ea39a135fc88408a362453a#diff-97ba6ea35c8dc52c65848399a1480a111e3fab8a0226ea4d7e26b0fbbd4560f0R128) (though confusingly the setting defaults to `true` on Android, which I don't love).

Reply:
By "the setting defaults to true on Android", do you mean this functions default: https://github.com/radarlabs/radar-sdk-android/blob/b94182de6c830c985f69c85121bbc9ca48a67b47/sdk/src/main/java/io/radar/sdk/RadarSettings.kt#L300 ?

Think some logic like that is why I had flipped it here, but I just changed it back.